content/applications/sales/sales/send_quotations/get_signature_to_validate.rst

```diff
@@ -1,51 +1,52 @@
-===================================
-Get a signature to confirm an order
-===================================
+=========================================
+Online signatures for order confirmations
+=========================================
 
-Online signatures are like electronic "fingerprints". By using them on Odoo, you will get
-automatic orders confirmation. You and your customers will save a lot of time by using this
-feature compared to a traditional process.
+The Odoo *Sales* application provides customers with the ability to confirm orders via an online
+signature directly on the sales order. Once the sales order is electronically signed by the
+customer, the salesperson attached to the sales order is automatically notified that the order is
+confirmed.
 
-Activate online signature
-=========================
+Activate online signatures
+==========================
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Online Signature**
-feature.
+To activate the *Online Signature* feature, go to :menuselection:`Sales app --> Configuration -->
+Settings`, scroll to the :guilabel:`Quotations & Orders` heading, and activate the
+:guilabel:`Online Signature` feature by checking the box beside it, and clicking the
+:guilabel:`Save` button in the top-left corner.
 
-.. image:: get_signature_to_validate/signature_1.png
+.. image:: get_signature_to_validate/signature-setting.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to enable online signature on Odoo Sales?
+   :alt: How to enable online signature on Odoo Sales.
```

Comment:
All three following images have the same alt text, its best for each to be descriptive and unique.
```suggestion
   :alt: How to enable online signature in Odoo Sales settings.
```